pyro/infer/autoguide/guides.py

```diff
@@ -721,10 +721,10 @@ class AutoNormalizingFlow(AutoContinuous):
     """
 
     def __init__(self, model, init_transform_fn):
+        super().__init__(model, init_loc_fn=init_to_feasible)
```

Comment:
Do I understand correctly that this failed because `self.transform` was a `TransformModule` and thus required `nn.Module.__init__()` to be called before setting that attribute?

Reply:
`transform` is just set to None in `__init__` to begin with, but this was needed to call `PyroModule.__init__` before setting `self._prototype_tensor`.